pkg/utils/fs/fs.go

```diff
@@ -248,3 +248,32 @@ func readDir(fs v1.FS, dirname string) ([]fs.DirEntry, error) {
 	sort.Slice(dirs, func(i, j int) bool { return dirs[i].Name() < dirs[j].Name() })
 	return dirs, nil
 }
+
+const (
+	Active   = "active_boot"
+	Passive  = "passive_boot"
+	Recovery = "recovery_boot"
+	LiveCD   = "livecd_boot"
+	Unknown  = "unknown"
+)
+
+// DetectBoot Here until we move this to sdk to accept a fs.
+func DetectBoot(fs v1.FS) (string, error) {
```

Comment:
This is currently on the sdk but the version in there does not accept a vfs so it always looks on the real system for the /proc/cmdline which makes it impossible to test properly